test/typeEvaluate.test.ts

```diff
@@ -899,9 +899,9 @@ t.test('deref with projection union', (t) => {
         },
         authorOrGhostName: {
           type: 'objectAttribute',
-          value: {
+          value: nullUnion({
             type: 'string',
-          },
+          }),
```

Comment:
Studio will never create a schema like this, no?

Reply:
Good point!

Reply:
Ah, this is not the studio schema, but the projected values. Given that `authorOrGhostName` can be optional, when you are projecting it will be a union of string and null. There shouldn't be any null nodes in the schema

Reply:
Aaah, I see now!